bedrock/firefox/templates/firefox/ios.html

```diff
@@ -122,6 +144,36 @@
   </div>{#-- /.container --#}
 </section>{#-- /#sync --#}
 
+{% if l10n_has_tag('tracking_protection') and waffle.switch('firefox-ios-global') %}
+<section class="section section-privacy" id="privacy">
+  <div class="container">
+    <h2 class="heading">{{ _('Keep your private info private') }}</h2>
+
+    <p class="lead">
+      {{ _('We believe your personal information belongs to you.') }}
+      {{ _('Firefox for iOS is filled with features to help keep it that way. ') }}
```

Comment:
We strip them on our side of tools (both reference and translations). The [current reference](https://github.com/mozilla-l10n/www.mozilla.org/blob/master/en-US/firefox/ios.lang#L124-L126) string has no trailing whitespaces but the demo pages looks fully localized right now.
https://www-demo1.allizom.org/it/firefox/ios/

If doesn't look like we strip them when we [store source](https://github.com/mozilla/bedrock/blob/master/lib/l10n_utils/dotlang.py#L70) (unlike we do for translations), maybe we should.